Apps/AdminWebClient/src/ClientApp/src/services/httpDelegate.ts

```diff
@@ -128,17 +128,6 @@ export default class HttpDelegate implements IHttpDelegate {
                     console.log(errorMessage);
                     return reject(errorMessage);
                 });
-            // TODO: Axios has bug with the delete method not using data fields.
```

Comment:
Lets keep TODOS until they are solved or dismissed

Reply:
created a bug https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/10506
Todos must be used for work in progress only and should not pass the code review/merged to dev

Reply:
> created a bug https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/10506
> Todos must be used for work in progress only and should not pass the code review/merged to dev

That sounds good as well. We can block PR if they have TODOs on them